Getting_Started.md

```diff
@@ -0,0 +1,128 @@
+
+Assuming that the dependencies above have been satisfied, you can now provision a Bunsen environment.  In order to deploy a bunsen environment, you need a few things first.
+
+1. First, we need a copy of this repository placed on a `controller` somewhere.
+    - This system is not part of the minimal environment specified above.
+2. Next, we need to generate an inventory file for ansible to utilize when provisioning the hosts.
+
+    An example inventory file
+    ```
+    ossbunsen-infra      ansible_user=vagrant ansible_python_interpreter=/usr/bin/python3 ansible_ssh_pass=vagrant
+    ossbunsen-resource   ansible_user=vagrant ansible_python_interpreter=/usr/bin/python3 ansible_ssh_pass=vagrant
+    ossbunsen-farm-f36-1 ansible_user=vagrant ansible_python_interpreter=/usr/bin/python3 ansible_ssh_pass=vagrant
+    
+    [infrastructure]
+    ossbunsen-infra
+    
+    [resources]
+    ossbunsen-resource
+    
+    [farms]
+    ossbunsen-farm-f36-1
+    
+    [all:vars]
+    deploying_account = admin
+    target_account = admin
+    ```
+
+    The `admin` account should be replaced with the account that you intend to do your development with.
+
+    NOTE: In place of `ansible_ssh_pass=vagrant`, you can use `ansible_ssh_private_key_file='/path/to/vagrantdir/.vagrant/machines/<name>/libvirt/private_key'` (of course if you're not using vagrant or you have another private key granted, then you should point at that instead)
+        - `/path/to/vagrantdir` would be where you cloned [the VDO vagrant example repository](https://github.com/dm-vdo/vagrant)
+
+    NOTE: You can also leave out `ansible_ssh_pass` and `ansible_ssh_private_key_file` and insted use `--ask-pass` when running the `ansible-playbook` commands below to be prompted for the passwords instead.
+
+3. Now we can run the provisioning for the first time.  It is generally recommended to provision the roles serially to ensure that any dependencies across roles are satisfied without the chance of a timing issue being introduced.
+
+    Example commands for each role.
+    ```
+    ansible-playbook -i inventory \
+                     -l infrastructure \
+                     provisioning/playbook.yml \
+                     --extra-vars="install_bunsen_support=true" \
+                     --ask-become-pass
+
+    ansible-playbook -i inventory \
+                     -l resources \
+                     provisioning/playbook.yml
+
+    ansible-playbook -i inventory \
+                     -l farms \
+                     provisioning/playbook.yml
+    ```
+
+    - Note the added options `--extra-vars="install_bunsen_support=true" --ask-become-pass` on the infrastructure role.
+        - `--extra-vars="install_bunsen_support=true"` instructs the ansible playbook to install the necessary COPR repositories on the `controller` system and install the python dependencies (python3-resource-discovery, and python3-utility-mill).  It is only needed to be specified one time.
```

Comment:
Line 93: I'd suggest replacing the last sentence with "It only needs to be specified one time".

Reply:
This will be in my next rev.

Reply:
This is addressed in my recent pushes.